MediaBrowser.Providers/TV/TheTVDB/TvDbClientManager.cs

```diff
@@ -24,24 +24,28 @@ public TvDbClientManager(IMemoryCache memoryCache)
         {
             _cache = memoryCache;
             _tvDbClient = new TvDbClient();
-            _tvDbClient.Authentication.AuthenticateAsync(TvdbUtils.TvdbApiKey);
-            _tokenCreatedAt = DateTime.Now;
         }
 
-        public TvDbClient TvDbClient
+        private TvDbClient TvDbClient
```

Comment:
Any specific reason for this to be private instead of public? Does that have any effect on plugins?

Reply:
Plugins dont have access to this class